__tests__/src/services/useAppConstants.test.ts

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useAppConstants } from '../../../src/services/useAppConstants';
+import supabaseClient from '../../../utils/supabaseClient';
+
+describe('useAppConstants tests', () => {
+
+    const mockFilterBuilder = {
+        eq: jest.fn(() => Promise.resolve({}))
+    };
```

Comment:
I think this function can be deleted.

Reply:
ok.  Let's consider creating an Util test class.